src/powershell/FinOpsToolkit.psm1

```diff
@@ -1,28 +1,144 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
+# Import the localized Data
+$script:localizedData = Import-LocalizedData -FileName 'FinOpsToolkit.strings.psd1' -BaseDirectory $PSScriptRoot
+
+#region Private functions
+<#
+    .SYNOPSIS
+        Creates a directory if it does not already exist.
+
+    .PARAMETER Path
+        Path to create directory.
+#>
+function New-Directory
+{
+    [CmdletBinding()]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [string]
+        $Path
+    )
+
+    if (-not (Test-Path -Path $Path))
+    {
+        Write-Verbose -Message ($LocalizedData.NewDirectory -f $Path)
+        $null = New-Item -ItemType 'Directory' -Path $Path
+    }
+}
+
+<#
+    .SYNOPSIS
+        Saves a released version of the FinOps hub bicep template to local disk.
+
+    .PARAMETER Version
+        Version of the FinOps hub to download. Defaults to latest.
+
+    .PARAMETER Destination
+        Path to store the download. Defaults to env:temp.
+
+    .PARAMETER Force
+        Optional. Will overwrite an existing file.
+
+    .EXAMPLE
+        Save-FinOpsHubTemplate
+
+        Downloads the latest version of FinOps hub template to current users' temp folder.
+
+    .EXAMPLE
+        Save-FinOpsHubTemplate -Version '1.0.0' -Destination 'C:\myHub' -Force
+
+        Downloads version 1.0.0 of FinOpsHub template to c:\myHub directory. It will overwrite an existing 1.0.0.zip file if it exists.
+#>
+function Save-FinOpsHubTemplate
+{
+    [CmdletBinding()]
+    param
+    (
+        [Parameter()]
+        [string]
+        $Version = 'latest',
+
+        [Parameter()]
+        [switch]
+        $Preview,
+
+        [Parameter()]
+        [string]
+        $Destination = $env:temp,
+
+        [Parameter()]
+        [switch]
+        $Force
+    )
+
+    New-Directory -Path $Destination
+    $releases = Get-FinOpsToolkitVersion -Latest:$($Version -eq 'Latest') -Preview:$Preview
```

Comment:
Is `$Version -eq 'Latest'` case-sensitive? The value you set it to is lowercase above.